cli/src/factory_ops/unboxing/init

```diff
@@ -367,6 +367,17 @@ echo "Ok." | tee -a ${LOG_FILE}
 # lock unboxing user
 lock_unboxing_user
 
+service_user=$(salt srvnode-2 pillar.get system:service_user:name --output=newline_values_only)
```

Comment:
@andkononykhin2 
Why is `salt srvnode-2` being used instead of `salt-call`?

Reply:
we performing that on node-1 but need to know the configuration of node-2, it's more accurate to refer to accordant node here instead of relaying on the same pillar for both nodes (that might become false in future).
Also the code above uses the salt `salt` calls a lot so I think it's safe to do the same here